src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/SetAzureRmDataLakeStoreItemAcl.cs

```diff
@@ -38,7 +38,7 @@ public class SetAzureDataLakeStoreItemAcl : DataLakeStoreFileSystemCmdletBase
         [ValidateNotNull]
         public DataLakeStorePathInstance Path { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
+        [Parameter(ValueFromPipeline = true, Position = 2, Mandatory = true,
```

Comment:
Is removeing the 'ByPropertyName' intentional?  If there are output items that have an Acl proeprty, this would eliminate some piping scenarios